### ISSUE TITLE
[Hooks] Showing "Enabling/Disabling all hooks for mod" even for mods that don't have hooks

### ISSUE THREAD

**Issue by Dreomite** [Vermintide-Modders/Vermintide-Mod-Framework#25]
```
[MOD][test_mod][INFO] (hooks): Enabling all hooks for mod: test_mod
```
```
[MOD][test_mod][INFO] (hooks): Disabling all hooks for mod: test_mod
```
shouldn't be shown for the mods that haven't specified a single hook.

**Comment by Azumgi** [comment]
Fixed in #30

**Comment by Dreomite** [comment]
Fixed in #30.
